students/ABartles/Exercise2/GridPrinter.py

```diff
@@ -0,0 +1,51 @@
+
+def gridprint(numrow=2,numcol=2, widthofcol=4, hightofcol=4):
```

Comment:
Generally want to separate the words in function names with an underscore: grid_print().  Also, keep an eye on spaces - keeping them consistent.